Makefile

```diff
@@ -282,37 +286,38 @@ fmt:
 	@echo "===> Formatting Go files <==="
 	@gofmt -s -l -w $(GO_FILES)
 
-.golangci-bin:
+$(GOLANGCI_LINT_BIN):
```

Comment:
I suppose we also need to update the Makefile in hack/netpol?

Reply:
I didn't update it because it's dead code that we should remove (https://github.com/antrea-io/antrea/issues/1740)